### PR TITLE
Additions to getting-started-troubleshooting.md

### PR DESCRIPTION
**Pull request by taishi8117**
This is an updated addition to getting-started-troubleshooting.md. Please ignore the previous pull requests #900 and #901 (already closed).
- Added fixation for an issue with resolving localhost.

### CODE REVIEW

File: website/content/docs/getting-started-troubleshooting.md [hunk 1]
```diff
@@ -63,6 +63,44 @@ INFO: Elapsed time: 1.883s.
...
+    ```bash
+    $ python -c "import socket; print socket.gethostbyname(socket.gethostname())"
+    Traceback (most recent call last):
+      File "<string>", line 1, in <module>
+    socket.gaierror: [Errno 8] nodename nor servname provided, or not known
+    ```
+    
+    If the output looks like a normal IP address, such as `127.0.0.1`, 
+    you don't have this issue.
+    If the output is similar to the above, you need to modify `/etc/hosts`
```
Comment (billonahill) [inline website/content/docs/getting-started-troubleshooting.md]: modify the `/etc/hosts` file

**Comment by taishi8117** [comment]
@billonahill fixed.

**Comment by billonahill** [comment]
👍

**Comment by objmagic** [comment]
👍

**Comment by maosongfu** [comment]
👍

File: website/content/docs/getting-started-troubleshooting.md [hunk 2]
```diff
@@ -63,6 +63,44 @@ INFO: Elapsed time: 1.883s.
 
     Then fix it correspondingly.
     
+* It is also possible that the host has an issue with resolving localhost.
+To check, run the following command in a shell. 
+
```
Comment (maosongfu) [inline website/content/docs/getting-started-troubleshooting.md]: Can u also add some sample errors output, so people can associate their errors with this trouble-shooting?

**Comment by taishi8117** [comment]
@maosongfu The sample error output is already written.

**Pull request merged**